CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
 
-We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, age, or religion.
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
```

Comment:
So you don't think a CoC is harmful, just unnecessary? Yeah, that's pretty much where I stand on this too.

That said, regardless of whether it's necessary or not, there's a pretty significant number of people here who seem to want a CoC. So either we waste a bunch of time creating one, or waste a bunch of time arguing against creating one. Given that I have no strong aversion to the idea of a CoC, I'll go with the former option.